app/packages/looker-3d/src/fo3d/point-cloud/use-pcd-material.tsx

```diff
@@ -58,14 +69,18 @@ export const usePcdMaterial = (
       return { min: 0, max: 1 };
     }
 
-    const intensity =
-      geometry.getAttribute("color") ?? geometry.getAttribute("intensity");
-
-    if (intensity) {
-      return computeMinMaxForColorBufferAttribute(intensity);
+    const isLegacyIntensity = !Boolean(geometry.getAttribute("intensity"));
+
+    if (isLegacyIntensity) {
+      const minMax = computeMinMaxForColorBufferAttribute(
```

Comment:
issue color, intensity are not required attributes of the PCD spec:

Min max can be 0,1 if the attribute is undefined. Although its not clear how to check if the attributes aren't defined here. The headers of the PCD file should define whats included, but I'm not sure what the loader actually does.